coa-helm/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: cromwell-on-azure
 description: A cromwell deployment designed to run on Azure with Postgres. 
-version: 0.2.187
+version: 0.2.188
```

Comment:
I think this might auto-bump via `Broadbot` :D

Reply:
That makes sense!  I'll undo that change